manuscript/setup/dev.md

```diff
@@ -93,10 +94,16 @@ Follow this section ONLY if you're NOT planning to use EKS, AKS, or GKE
 
 ```bash
 # Install `yq` CLI from https://github.com/mikefarah/yq
+# 'yq' is a lightweight and portable command-line YAML processor
+
 # Install `jq` CLI from https://stedolan.github.io/jq/download
+# 'jp' is a lightweight and portable command-line JSON processor
 
 kubectl create namespace dev
 
+# Helm is a package manager for Kubernetes applications
+# Traefik is a reverse proxy server and load balancer that handles
+#   traffic that is first entering your Kubernetes cluster
```

Comment:
I wouldn't say "first" but rather "traffic from outside the cluster" (e.g., browser, CLI, apps running outside that cluster, etc.).